src/smt/theory_str_noodler/decision_procedure.h

```diff
@@ -267,29 +278,29 @@ namespace smt::noodler {
         std::unordered_set<BasicTerm> init_length_sensitive_vars;
         Formula formula;
         AutAssignment init_aut_ass;
+        // contains to/from_code/int transformations
+        std::vector<std::tuple<BasicTerm,BasicTerm,TransformationType>> transformations;
 
         // the length formula from preprocessing, get_lengths should create conjunct with it
         LenNode preprocessing_len_formula = LenNode(LenFormulaType::TRUE,{});
+        std::vector<LenNode> disequations_len_formula_conjuncts;
```

Comment:
Add a comment explaining what this is for and how it is used.

Reply:
I added something

Reply:
Looks good to me.